src/main/g8/src/main/scala/$package$/Layers.scala

```diff
@@ -1,22 +1,78 @@
 package $package$
 
-import slick.jdbc.JdbcProfile
-import slick.jdbc.PostgresProfile
-
 import zio._
-import zio.logging.backend.SLF4J
 
 object Layers {
 
-  object JdbcProfile {
-    val live: ULayer[JdbcProfile] = ZLayer.succeed[JdbcProfile](PostgresProfile)
-  }
-
   object Logger {
+    import zio.logging.backend.SLF4J
+
     val live: ULayer[Unit] = ZLayer.make[Unit](
       Runtime.removeDefaultLoggers,
       SLF4J.slf4j
     )
   }
 
+$if(enable_slick.truthy)$
+  object Repository {
+    import com.typesafe.config.Config
+    import slick.interop.zio.DatabaseProvider
+    import slick.jdbc.JdbcProfile
+    import slick.jdbc.PostgresProfile
+    import $package$.infrastructure.slick.{ SlickHealthCheckService, SlickItemRepository }
+    import $package$.api.healthcheck.HealthCheckService
+    import $package$.domain.ItemRepository
```

Comment:
Can we make these a top level imports? It's fine if they are enclosed in another `enable_slick.truthy` test.

Reply:
done